go.sum

```diff
@@ -925,6 +925,8 @@ github.com/lightstep/lightstep-tracer-common/golang/gogo v0.0.0-20190605223551-b
 github.com/lightstep/lightstep-tracer-go v0.18.1/go.mod h1:jlF1pusYV4pidLvZ+XD0UBX0ZE6WURAspgAczcDHrL4=
 github.com/linode/linodego v1.1.0 h1:ZiFVUptlzuExtUbHZtXiN7I0dAOFQAyirBKb/6/n9n4=
```

Comment:
You will need to run go mod tidy

Reply:
Thanks! Just saw this, will be fixed in a moment. 👍

Reply:
Adjusted so the `go mod tidy` run is part of the build update commit